### ISSUE TITLE
Loading indicator for Auto-Type unlock?

### ISSUE THREAD

**Issue by dan-el** [strongbox-password-safe/Strongbox#200]
This is just a cosmetic issue, not impairing functionality.

Steps to reproduce the "issue":

1) Open browser app (e.g. Chrome) and navigate to a login form on a website
2) Press the "Passwords" button on the keyboard to initiate Strongbox Auto-Type
3) TouchID appears to unlock the database
4) Use TouchID

Here I think it would be nice to see a loading indicator rather than only the list of databases.

**Comment by mmcguill** [comment]
Yes, notice this issue myself. Basically the library I use to show the spinner/progress indicator doesn't play well with App Extensions, so I need to find a replacement.
